src/panels/TestPanel.tsx

```diff
@@ -0,0 +1,20 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
Maybe place all the examples files in one folder in the localization folder? That way we will have an easy time removing them when we dont need them anymore

Reply:
Moving the TestPanel at least but the translation files must be in the locale folder otherwise the backend won't find them